client/signup/steps/design-picker/index.jsx

```diff
@@ -250,76 +254,65 @@ class DesignPickerStep extends Component {
 		return undefined;
 	}
 
-	render() {
-		const { flowName, stepName, userLoggedIn, isReskinned, isMobile, translate } = this.props;
-		const { selectedDesign } = this.state;
-
-		if ( selectedDesign ) {
-			const isBlankCanvas = isBlankCanvasDesign( selectedDesign );
-			const designTitle = isBlankCanvas ? translate( 'Blank Canvas' ) : selectedDesign.title;
-			const defaultDependencies = { selectedDesign };
-			const locale = ! userLoggedIn ? getLocaleSlug() : '';
-
-			return (
-				<StepWrapper
-					{ ...this.props }
-					className="design-picker__preview"
-					fallbackHeaderText={ designTitle }
-					headerText={ designTitle }
-					fallbackSubHeaderText={ '' }
-					subHeaderText={ '' }
-					stepContent={ this.renderDesignPreview() }
-					align={ isMobile ? 'left' : 'center' }
-					hideSkip
-					hideNext={ false }
-					nextLabelText={ translate( 'Start with %(designTitle)s', {
-						args: { designTitle },
-					} ) }
-					defaultDependencies={ defaultDependencies }
-					backUrl={ getStepUrl( flowName, stepName, '', locale ) }
-					goToNextStep={ this.submitDesign }
-					stepSectionName={ designTitle }
-				/>
-			);
-		}
+	const isMobile = useViewportMatch( 'small', '<' );
+
+	const { flowName, stepName, isReskinned } = props;
```

Comment:
I personally prefer to do object destructuring at the beginning of the component.

Reply:
You're right that's more usual. Just a hangover from refactoring the `render()` method I guess.